gobblin-runtime/src/main/java/gobblin/runtime/limiter/LimitingExtractorReporter.java

```diff
@@ -0,0 +1,12 @@
+package gobblin.runtime.limiter;
+import gobblin.configuration.State;
+import gobblin.util.limiter.Limiter;
+
+/**
+ * This interface provides the capability to report events or metrics when
+ * the {@link Limiter} inside of {@link gobblin.runtime.LimitingExtractorDecorator}
+ * has been reached.
+ */
+public interface LimitingExtractorReporter {
```

Comment:
This has been removed. As a replacement, a key list which needs to be reported will be added to WorkUnit and retrieved from LimiterExtractorReporter when acquirePermit() gets failed.